xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepository.java

```diff
@@ -50,6 +50,16 @@ public void deleteById(String aspspId) {
         }
     }
 
+    @Override
+    public void deleteAll() {
+        IndexWriterConfig indexWriterConfig = new IndexWriterConfig();
+        try (IndexWriter indexWriter = new IndexWriter(directory, indexWriterConfig)) {
+            indexWriter.deleteAll();
```

Comment:
could you please consult with Artem what is the best impl here, delete records from the index or delete the whole index

Reply:
Artem said this approach is good enough, however, there are other remarks from him that I need to take care of.